src/WebJobs.Script/Config/HostHealthMonitorConfiguration.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class HostHealthMonitorConfiguration
+    {
+        internal const float DefaultCounterThreshold = 0.80F;
+
+        public HostHealthMonitorConfiguration()
+        {
+            Enabled = true;
+
+            // these default numbers translate to a 50% health rating
+            // over the window.
+            HealthCheckInterval = TimeSpan.FromSeconds(15);
```

Comment:
@paulbatum made a comment on the issue https://github.com/Azure/azure-webjobs-sdk-script/issues/2115 that we may want to adjust these default numbers.  I chose a longer time period because I wanted to give the host a chance to heal via periodic restart checks.